brayns/engine/Scene.cpp

```diff
@@ -257,13 +255,124 @@ void Scene::visitModels(const std::function<void(Model&)>& functor)
         functor(modelDescriptor->getModel());
 }
 
-void Scene::buildDefault()
+void Scene::buildGrid()
```

Comment:
Since we now have a few demo scenes maybe we should move these builders into a new file?